### DISCUSSION TITLE
cannot acces to my website, error 404

### DISCUSSION THREAD

**Discussion by NathanPoire** [community/community#50514] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello, I have got an issue and I don't know how to fix it. When I try to open my website, I got an error 404
https://nathanpoire.github.io/my-final-profile/
But I am sure I made everything correct. I have a html and css profile. I can open my profile on chrome, it isnt working just on github.
Thanks for your answers,
Nathan

**Comment by meppu** [comment]
This is because your folder contents are inside folder `my-final-project/` and not in root (`/`) directory. As you can see following links works:
- https://nathanpoire.github.io/my-final-profile/my-final-profile/

Just unpack your contents into root directory instead of putting them into `my-final-project/`

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @NathanPoire ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
